src/LoRa.cpp

```diff
@@ -607,9 +607,21 @@ void LoRaClass::setOCP(uint8_t mA)
   writeRegister(REG_OCP, 0x20 | (0x1F & ocpTrim));
 }
 
+void LoRaClass::LoRandomSeed() {
+  writeRegister(REG_OP_MODE, 0x72);
+  writeRegister(REG_MODEM_CONFIG_1, 0x72);
+  writeRegister(REG_MODEM_CONFIG_2, 0x70);
+}
+
 byte LoRaClass::random()
 {
-  return readRegister(REG_RSSI_WIDEBAND);
+  uint8_t x = 0;
+  for (uint8_t j = 0; j < 8; j++) {
+    x += (readRegister(REG_RSSI_WIDEBAND) & 0x01);
+    x = x << 1;
+    delay(1);
```

Comment:
Is a delayed needed here? Does it need to be exactly 1 ms?

Reply:
Not necessarily. I added it to put a little more entropy.